.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+For bug reports, please respect the following guidelines and check the boxes accordingly.
```

Comment:
In my experience badly written bug reports are mostly useless, so I'd prefer to keep this as it is.